paris/src/test/java/com/airbnb/paris/WithSubStyleView.java

```diff
@@ -0,0 +1,34 @@
+package com.airbnb.paris;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+
+import com.airbnb.paris.annotations.Styleable;
+import com.airbnb.paris.annotations.StyleableChild;
+
+@Styleable("Test_WithSubStyleView")
+public class WithSubStyleView extends View {
```

Comment:
should you have a view in kt as well to test that usage?

Reply:
I had originally and ran into an issue because the annotated field was private. I assume because it was trying to access the backing field? I'm going to have to spend some time figuring out how to make the annotation processor work well for both Java and Kotlin classes. Seems like it could get tricky.